src/ComputeSharp.D2D1/Attributes/Enums/D2D1ChannelDepth.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ComputeSharp.D2D1;
+
+/// <summary>
+/// Indicates the channel depth of a stage in a Direct2D rendering pipeline.
+/// </summary>
+/// <remarks>
+/// <para>This type exposes the available values in <see href="https://docs.microsoft.com/windows/win32/api/d2d1effectauthor/ne-d2d1effectauthor-d2d1_channel_depth"/>.</para>
+/// <para>For more info, see <see href="https://docs.microsoft.com/windows/win32/direct2d/precision-and-clipping-in-effect-graphs"/>.</para>
+/// </remarks>
+public enum D2D1ChannelDepth
+{
+    /// <summary>
+    /// The channel depth is the default, and it will be inherited by the inputs.
+    /// </summary>
+    Default = 0,
```

Comment:
recommend mapping to D2D's consts instead of `0`, `1`, `2`

e.g. 
<img width="310" alt="image" src="https://user-images.githubusercontent.com/10873410/167271645-f444891f-1cf8-493d-986e-28f3f397bbaa.png">

Reply:
Fixed in d6753ecaed2b9ae5d99a6c8c748ca18d00de3e8c.